src/controllers/categories.js

```diff
@@ -0,0 +1,22 @@
+const { CategoriesService } = require('../services');
+const categoriesService = new CategoriesService();
```

Comment:
Make this as private field of CategoriesController class (#categoriesService = new CategoriesService())

Reply:
Applied.